package-lock.json

```diff
@@ -1,8 +1,3958 @@
 {
   "name": "sway-vscode-plugin",
   "version": "0.2.4",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
This is a surprising amount of changes to `package-lock.json`. When I pull down this branch's `package.json` and run `npm install`, I get a much smaller diff to `package-lock.json`. Do you know where all these changes come from?